src/Validator/Text.php

```diff
@@ -23,21 +23,33 @@
  */
 class Text extends Validator
 {
+    const NUMBERS = [ '0','1','2','3','4','5','6','7','8','9' ];
+    const ALPHABET_UPPER = [ 'A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z' ];
+    const ALPHABET_LOWER = [ 'a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z' ];
+
     /**
      * @var int
      */
     protected int $length = 0;
 
+    /**
+     * @var string[]
+     */
+    protected array $whitelist = [];
```

Comment:
we want to gradually move away from the convention of using `whitelist` in favour of `allowList`

Maybe this PR can be a start to that. WDYT ?